BusBoard.ConsoleApp/Program.cs

```diff
@@ -18,32 +18,57 @@ static void Main(string[] args)
     {
       ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
 
-      var clientJSONResponse = getClientJSONResponse();
-      List<BusData> listArrivingBuses = getArrivingBussesFromJSON(clientJSONResponse);
-      printBusesToConsole(listArrivingBuses);
+      userConsoleInteration();
+    }
+
+    private static void userConsoleInteration()
+    {
+      var userInput = "";
+      while (userInput != "Exit")
+      {
+        Console.WriteLine("\n Please enter an instruction: \n" +
+                          "1.The bus stop code you wish to query (490008660N)\n" +
+                          "2 Exit \n");
+
+        userInput = Console.ReadLine();
+        if (userInput != "Exit")
+        {
+          var clientJSONResponse = getClientJSONResponse(userInput);
+          if (clientJSONResponse.StatusCode == HttpStatusCode.OK)
+          {
+            List<BusData> listArrivingBuses = getArrivingBusesFromJSON(clientJSONResponse);
+            printBusesToConsole(listArrivingBuses);
+          }
+          else
+          {
+            Console.WriteLine($"Invalid bus stop code: {userInput}");
+          }
+        }
+      }
     }
 
     private static void printBusesToConsole(List<BusData> listArrivingBuses)
     {
       Console.WriteLine($"The next 5 buses arriving at {listArrivingBuses[0].StationName} are: \n");
       foreach (var bus in listArrivingBuses)
       {
-        Console.WriteLine(bus.VehicleID);
+        Console.WriteLine($"Bus {bus.VehicleID} on line {bus.LineID} to {bus.DestinationName} will be arriving in {bus.TimeToStationInMin} minutes. Expected: {bus.ExpectedArrivalTime}");
       }
     }
 
 
-    private static List<BusData> getArrivingBussesFromJSON(IRestResponse clientJSONResponse)
+    private static List<BusData> getArrivingBusesFromJSON(IRestResponse clientJSONResponse)
     {
       List<BusData> listArrivingBusDataFromJSON = JsonConvert.DeserializeObject<List<BusData>>(clientJSONResponse.Content);
+      List<BusData> orderedListArrivingBusData = listArrivingBusDataFromJSON.OrderBy(o=>o.ExpectedArrivalTime).ToList();
```

Comment:
I think we only want to see the first 5 buses so you might want a `Take` method here too